tests/test_app.py

```diff
@@ -0,0 +1,235 @@
+import datetime
+from unittest.mock import patch
+
+from freezegun import freeze_time
+from page_analyzer.utils.db import (
+    create_check,
+    create_url,
+    get_url_by_name,
+    get_url_checks,
+    get_url_data,
+)
+
+
+def test_index(client):
+    # given
+    route = "/"
+
+    # when
+    response = client.get(route)
+
+    # then
+    assert response.status_code == 200
+    assert "Бесплатно проверяйте сайты на SEO-пригодность" in response.text
+
+
+def test_add_url(test_database, client):
+    # given
+    url_name = "https://www.example.com"
+    route = "/urls"
+
+    # when
+    response = client.post(route, data={"url": url_name}, follow_redirects=True)
+
+    # then
+    url_id = get_url_by_name(url_name)
+    assert url_id == 1
+    assert response.request.path == "/urls/1"
+    assert "Страница успешно добавлена" in response.text
```

Comment:
it's better to do response checks first and then check the side effects (DB entry creation)